scheduler/src/cook/kubernetes/api.clj

```diff
@@ -510,7 +510,10 @@
   See https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container"
   [clobber-synthetic-pods node-name->pods pool-name]
   (->> node-name->pods
-       (filter first) ; Keep those with non-nil node names.
+       ; Keep those with non-nil node names
+       (filter first)
+       ; Keep those with non-nil and non-empty pods
```

Comment:
Worth saying that we can see these early in a node's lifetime. (As in, this code is to protect from a particular landmine that occurs in a particular place)

Reply:
Good call, done